composer.json

```diff
@@ -6,14 +6,32 @@
   "license": "mit",
   "require": {
     "php": "^8.0.2",
-    "craftcms/cms": "^4.0.0"
+    "craftcms/cms": "^4.0.0",
+    "putyourlightson/craft-sprig": "^2.0.0"
```

Comment:
Why is the Sprig plugin being required here?

Reply:
Sprig Core does not currently work without the Sprig plugin (it references a class from the Sprig plugin). If your intention is that it should work without the Sprig plugin, you can remove that dependency and fix the error that PHPstan reports.

Reply:
Yeah, it is. Requiring the plugin defeats the purpose of having Sprig core, but having it as a dev requirement should be fine.

Reply:
Sure, put it in `require-dev` if you like!